src/main/java/net/glowstone/entity/passive/GlowCow.java

```diff
@@ -19,6 +21,19 @@ public GlowCow(Location location) {
     }
 
     @Override
+    public void kill() {
+        super.kill();
+
+        Random r = new Random();
+
+        int beefCnt = r.nextInt(3);
+        int leatherCnt = r.nextInt(2);
+
+        getWorld().dropItemNaturally(getLocation(), new ItemStack(Material.RAW_BEEF, beefCnt));
```

Comment:
Well, entities die due to damage due to a DamageCause, but blocks get destroyed due to a block break packet with a certain held item.